Building the default b2d ISO fails

$ make deploy/iso/minishift.iso

    Step 13 : RUN /make_iso.sh
     ---> Running in cc348157ad10
     /bin/sh: 1: /make_iso.sh: not found
    The command '/bin/sh -c /make_iso.sh' returned a non-zero code: 127
    make: *** [deploy/iso/minishift.iso] Error 1

--- a/deploy/iso/Dockerfile
+++ b/deploy/iso/Dockerfile
@@ -42,7 +42,7 @@ RUN curl -fSL -o /tmp/dockerbin.tgz https://get.docker.com/builds/Linux/x86_64/d
 RUN $ROOTFS/usr/local/bin/docker -v
 
 # Build the actual iso image
-RUN /make_iso.sh
+RUN /tmp/make_iso.sh
 
 # Output it to transfer back to the host
 CMD ["cat", "boot2docker.iso"]

